apps/ledger-live-mobile/src/screens/Swap/Form/Modal/Confirmation.tsx

```diff
@@ -17,14 +18,13 @@ import { addPendingOperation, getMainAccount } from "@ledgerhq/live-common/accou
 import { AccountLike, DeviceInfo, Operation, SignedOperation } from "@ledgerhq/types-live";
 import { Device } from "@ledgerhq/live-common/hw/actions/types";
 import { postSwapAccepted, postSwapCancelled } from "@ledgerhq/live-common/exchange/swap/index";
-import { getEnv } from "@ledgerhq/live-env";
 import { InstalledItem } from "@ledgerhq/live-common/apps/types";
+import { useBroadcast } from "@ledgerhq/live-common/hooks/useBroadcast";
 import { renderLoading } from "../../../../components/DeviceAction/rendering";
 import { updateAccountWithUpdater } from "../../../../actions/accounts";
 import DeviceAction from "../../../../components/DeviceAction";
 import QueuedDrawer from "../../../../components/QueuedDrawer";
 import ModalBottomAction from "../../../../components/ModalBottomAction";
-import { useBroadcast } from "../../../../components/useBroadcast";
```

Comment:
does it need to be removed from the mobile project then?

Reply:
Not sure to understand why. I just moved the import upper in the file and change the import path.
The `useBroadcast` file has been moved to live-common and the desktop one has been deleted.